inst/examples/03-formats.Rmd

```diff
@@ -218,9 +218,195 @@ favicon: "favicon.ico"
 
 A nice effect of setting `description` and `cover-image` is that when you share the link of your book on some social network websites such as Twitter, the link can be automatically expanded to a card with the cover image and description of the book.
 
+### Bootstrap 4 style
```

Comment:
I now understand why the current chapter `Bootstrap style` was conflicting.

We could maybe rename the other chapter if you would have a preference of naming this one `Bootstrap style`

But this is title is ok to me. 
I would just add an id. 
```suggestion
### Bootstrap 4 style {#bs4-book}
```